docs/blocks/custom-fluids.md

```diff
@@ -0,0 +1,851 @@
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~ ~1 air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~ ~ ~1 fluids:fluid_template3" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template1') || query.block_neighbor_has_any_tag(0, 0, 1, 'template1') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template1') || query.block_neighbor_has_any_tag(0, 0, -1, 'template1')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~ ~-1 air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~ ~ ~-1 fluids:fluid_template3" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template1') || query.block_neighbor_has_any_tag(0, 0, 1, 'template1') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template1') || query.block_neighbor_has_any_tag(0, 0, -1, 'template1')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~1 ~ ~ air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~1 ~ ~ fluids:fluid_template3" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template1') || query.block_neighbor_has_any_tag(0, 0, 1, 'template1') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template1') || query.block_neighbor_has_any_tag(0, 0, -1, 'template1')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~-1 ~ ~ air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~-1 ~ ~ fluids:fluid_template3" ]
+            }
+          },
+
+          {
+            "condition": "!query.block_neighbor_has_any_tag(1, 0, 0, 'template1') && !query.block_neighbor_has_any_tag(0, 0, 1, 'template1') && !query.block_neighbor_has_any_tag(-1, 0, 0, 'template1') && !query.block_neighbor_has_any_tag(0, 0, -1, 'template1')",
+            "run_command": {
+              "target": "self",
+              "command": [ "setblock ~ ~ ~ air" ]
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+</Spoiler>
+
+## Outer Fluid Block 3
+
+Below is the JSON for the `outer fluid block 3`. To replicate the json, copy and quick replace `template` with your fluid's name. This block solely places the falling fluid block. Moreover, all fluid blocks check for the existence of at least one fluid block from a higher tier next to them. If none is found, the block deletes itself.
+
+<CodeHeader>BH\blocks\fluid_template\fluid_template3.json</CodeHeader>
+<Spoiler title="Outer Fluid 3 JSON">
+
+```json
```

Comment:
Anyway we can use the latest `1.19.60` format version? Also shorten all query to q